[ICDS Dashboard]Identifying the areas in the Dashboard where the height eligiblity is 6mnth to 5 years and not 0-5 years

ShubhamBansal1997 opened the pull request:
<!--- Provide a link to the ticket or document which prompted this change -->

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Identifying the areas in the Dashboard where the height eligiblity is 6mnth to 5 years and not 0-5 years
https://app.asana.com/0/0/1153564689437801/f
##### FEATURE FLAG
<!--- If this is specific to a feature flag, which one? -->

##### PRODUCT DESCRIPTION
<!--- For non-invisible changes, describe user-facing effects. -->

ShubhamBansal1997 committed:
[fix]: change the height eligibilty

- Github Issue:

Authored-by: Shubham Bansal <shubhambansal17@hotmail.com>
Signed-off-by: Shubham Bansal <shubhambansal17@hotmail.com>

ShubhamBansal1997 committed:
[fix]: fix the agg test

- Github Issue:

Authored-by: Shubham Bansal <shubhambansal17@hotmail.com>
Signed-off-by: Shubham Bansal <shubhambansal17@hotmail.com>

Rohit25negi reviewed (approved):
Looks good. 
There are other PRs as well no? I could not find them could you please add those to the ticket. Just for my and other's reference.

Rohit25negi commented on custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py:
```diff
@@ -90,7 +90,7 @@ def staging_queries(self):
             # age_in_months_start & age_in_months_end in UCR
             ('height_eligible',
-                "SUM(CASE WHEN chm.age_in_months >= 6 AND chm.age_tranche NOT IN ('72') AND "
+                "SUM(CASE WHEN chm.age_in_months >= 0 AND chm.age_tranche NOT IN ('72') AND "
```
I think you could just remove the >=0 clause. That would just make the statement short

Rohit25negi commented on custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py:
```diff
@@ -99,7 +99,7 @@ def _state_aggregation_query(self, state_id):
             valid_in_month, end_month_string, start_month_string
         )
-        height_eligible = "({} AND {} > 6 AND {} <= 60)".format(valid_in_month, age_in_months_end, age_in_months)
+        height_eligible = "({} AND {} >= 0 AND {} <= 60)".format(valid_in_month, age_in_months_end, age_in_months)
```
same comment.

Rohit25negi reviewed (changes requested):
Looks good. Left a few code clearity comments.
Also we need to track that this need historical data correction.

ShubhamBansal1997 committed:
[refactor]: remove extra constraint which is already accounted in the next
condition

- Github Issue:

Authored-by: Shubham Bansal <shubhambansal17@hotmail.com>
Signed-off-by: Shubham Bansal <shubhambansal17@hotmail.com>

ShubhamBansal1997 committed:
[fix]: updated icds aggregration script

- Github Issue:

Authored-by: Shubham Bansal <shubhambansal17@hotmail.com>
Signed-off-by: Shubham Bansal <shubhambansal17@hotmail.com>

Rohit25negi reviewed (approved):
Looks good. Thanks
